packages/dds/tree/src/feature-libraries/deltaUtils.ts

```diff
@@ -22,7 +22,7 @@ export function mapFieldMarks<TIn, TOut>(
     fields: Delta.FieldMarks<TIn>,
     func: (tree: TIn) => TOut,
 ): Delta.FieldMarks<TOut> {
-    const out: Delta.FieldMarks<TOut> = new Map();
+    const out: Map<FieldKey, Delta.MarkList<TOut>> = new Map();
```

Comment:
one way around having to name the type here is to do functional style construction (either via some map over map helper, or building the key value pair list an passing it to the map constructor. I'm fine with the code how it is, just wanted to mention the option (i'm not sure it would be better)